util/settings/settings.go

```diff
@@ -1740,7 +1757,7 @@ func (a *ArgoCDSettings) IsSSOConfigured() bool {
 }
 
 func (a *ArgoCDSettings) IsDexConfigured() bool {
-	if a.URL == "" {
+	if a.URL == "" && len(a.URLs) == 0 {
```

Comment:
Is Dex using this new URLs setting? It seems unclear if Dex should be valid if there are only URLs configured.